examples/form/index.html

```diff
@@ -64,9 +64,18 @@ <h2 class="ds-c-alert__heading">1 error with your application</h2>
         <label class="ds-u-display--inline-block" for="input-no">No</label>
       </fieldset>
 
-      <input class="ds-c-button ds-c-button--primary ds-u-margin-top--3" type="submit" value="Report issue" />
+      <button class="ds-c-button ds-c-button--primary ds-u-margin-top--3" type="submit">
+        <span id="spinner" class="ds-c-spinner ds-u-display--none"></span>
+        Report issue
+      </button>
       <a class="ds-c-button" href="#">Cancel</a>
     </form>
   </div>
+  <script>
+    document.querySelector("form").addEventListener("submit", function(event) {
+      event.preventDefault();
+      document.querySelector("#spinner").classList.toggle("ds-u-display--none");
```

Comment:
Just flagging that we should apply whatever a11y best practices here...once we figure that out.